docs/extending.md

```diff
@@ -382,20 +382,20 @@ To create a new route, populate the `routes` section with the corresponding entr
 | **id**        | Unique identifier.                                                                     |
 | **path**      | Runtime path of the route.                                                             |
 | **component** | The main [component](#components) to use for the route.                                |
-| layout        | The layout [component](#components) to use for the route.                              |
-| auth          | List of [authentication guards](#authentication-guards). Defaults to `[ "app.auth" ]`. |
-| data          | Custom property bag to carry with the route.                                           |
+| **layout**    | The layout [component](#components) to use for the route.                              |
```

Comment:
the bold font identifies the `required` properties, while regular is for optional.